Source/Magick.NET.CrossPlatform/ubuntu.16.04/Dockerfile

```diff
@@ -2,20 +2,49 @@ FROM ubuntu:16.04
 
 # Install packages
 RUN apt-get update
-RUN apt-get install -y gcc g++ make autoconf libjpeg-turbo8-dev git openssh-server
+RUN apt-get install -y gcc g++ make autoconf autopoint pkg-config libtool nasm git openssh-server
 
 # Initialize the sshd server
-RUN mkdir /var/run/sshd;chmod 755 /var/run/sshd
+RUN mkdir /var/run/sshd; chmod 755 /var/run/sshd
 RUN sed -i -e 's/#PasswordAuthentication yes/PasswordAuthentication no/g' /etc/ssh/sshd_config
-RUN mkdir ~/.ssh;echo "ssh-rsa AAAAB3NzaC1yc2EAAAABJQAAAQEAwFyerjwW+MpAIV056qHP0pJLAJn2acRQNtFnmQiCXP6AukRMr7rJ23w3TsyGEKzTGCi/jQMHTWzGYGj7OtNNTN+OSFvx9OVg+4wr6kYPQmPZBnPvUg727nqggAGZ90j7bL/B+7qiUJxoHOBqPSS56A7oMmsA19Iy2kGaYlBLB0J+YY1wHlRyVa4vyxZox6n9ORZ6Wat+OGc6fyo9sNYWBCVzufUWAhQM2JCctCO5htglytD3vk/wJnJ5k/VEajYueM90m3V7fv4DucWvPJk75TlSJUkh81yujQttPmar74k4vcTf/zfUpkcYBHucvMATUrBfpE+e1OL013JM470RuQ==" > ~/.ssh/authorized_keys;chmod 600 ~/.ssh/authorized_keys
+COPY authorized_keys /root/.ssh/authorized_keys
+RUN chmod 600 ~/.ssh/authorized_keys
+
+# Build libbz2
+RUN cd ~; wget -q http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz; tar xf bzip2-1.0.6.tar.gz;
```

Comment:
I wonder if we could use the repositories we have in the ImageMagick repo here? For example: https://github.com/ImageMagick/bzlib. I do think I broke some of them under Linux for the Windows build but it would be nice if we could keep them in sync with those repos. And we could always take the most recent checkout.